robot_sort/robot_sort.py

```diff
@@ -81,24 +81,49 @@ def set_light_on(self):
         Turn on the robot's light
         """
         self._light = "ON"
+
     def set_light_off(self):
         """
         Turn off the robot's light
         """
         self._light = "OFF"
+
     def light_is_on(self):
         """
         Returns True if the robot's light is on and False otherwise.
         """
         return self._light == "ON"
 
     def sort(self):
-        """
-        Sort the robot's list.
-        """
-        # Fill this out
-        pass
 
+        # * TURN ON ROBOT
+        self.set_light_on()
+        # * CURRENT ITEM IS NONE, SWAP FIRST.
+        self.swap_item()
+
+        # ? PRINT AND CHECK VALUES
+        print(self._item)
+
+        # * WHILE TRUE
```

Comment:
It's best to avoid comments that are redundant. In this case the comment does not add enough to the understanding of what the code does to warrant it. Better to explain _why_ this code is here rather than what it's doing.